src/Backend/OnboardingInsuranceAPI/Program.cs

```diff
@@ -23,7 +27,8 @@ static async Task Main()
             {
                 services
                     .AddScopedByInterface<IHandler>()
-                    .AddDbContext<DataContext>(); //AddDbContextFactory try booth
+                    .AddCosmosDb(); //!!!PRODUCTION DB!!!
```

Comment:
It probably isn't production db, as it depends on you configuration. Now you are more or less in "dev". "prod" would be different db. Not in your case obviously as you don't have prod environment. But the difference is not testing/production.